Gopkg.lock

```diff
@@ -1520,6 +1560,30 @@
     "kubernetes/typed/storage/v1",
     "kubernetes/typed/storage/v1alpha1",
     "kubernetes/typed/storage/v1beta1",
+    "listers/admissionregistration/v1alpha1",
+    "listers/admissionregistration/v1beta1",
+    "listers/apps/v1",
+    "listers/apps/v1beta1",
+    "listers/apps/v1beta2",
+    "listers/autoscaling/v1",
+    "listers/autoscaling/v2beta1",
+    "listers/batch/v1",
+    "listers/batch/v1beta1",
+    "listers/batch/v2alpha1",
+    "listers/certificates/v1beta1",
+    "listers/core/v1",
+    "listers/events/v1beta1",
+    "listers/extensions/v1beta1",
+    "listers/networking/v1",
+    "listers/policy/v1beta1",
+    "listers/rbac/v1",
+    "listers/rbac/v1alpha1",
+    "listers/rbac/v1beta1",
+    "listers/scheduling/v1alpha1",
+    "listers/settings/v1alpha1",
+    "listers/storage/v1",
+    "listers/storage/v1alpha1",
+    "listers/storage/v1beta1",
```

Comment:
Possibly more a question for @ldez: Is it normal that dep references packages which are not used? I thought dep only pulls in what is really needed.

Reply:
These are all supposed to be direct dependencies when compiling `informerFactory`. 
[https://github.com/kubernetes/kubernetes/blob/aa02c0f5191af40cbf7a18a8beec24c56ed7b860/staging/src/k8s.io/client-go/informers/factory.go#L42](https://github.com/kubernetes/kubernetes/blob/aa02c0f5191af40cbf7a18a8beec24c56ed7b860/staging/src/k8s.io/client-go/informers/factory.go#L42)
[https://github.com/kubernetes/kubernetes/blob/aa02c0f5191af40cbf7a18a8beec24c56ed7b860/staging/src/k8s.io/client-go/informers/core/v1/pod.go#L30](https://github.com/kubernetes/kubernetes/blob/aa02c0f5191af40cbf7a18a8beec24c56ed7b860/staging/src/k8s.io/client-go/informers/core/v1/pod.go#L30)

Reply:
👌